src/bitcoin-tx.cpp

```diff
@@ -794,7 +793,7 @@ static int CommandLineRawTx(int argc, char* argv[])
             // param: hex-encoded bitcoin transaction
             std::string strHexTx(argv[1]);
             if (strHexTx == "-")                 // "-" implies standard input
-                strHexTx = readStdin();
+                strHexTx = TrimString(readStdin());
```

Comment:
This one actually *should* be locale-dependent.

Reply:
@luke-jr I'm not sure I follow: could you explain why? Please include an example if possible where the new code would give bad results.

Reply:
I'm not sure how important this specific is, but generally we do not use locale-dependent input handling in bitcoind nor utilities. We'd like to avoid differences in locale resulting in hard-to-reproduce bugs and unexpected behavior. Note that this tool is mainly for scripting use where deterministic behavior is important.